dotCMS/src/main/java/com/dotcms/graphql/business/PageAPIGraphQLTypesProvider.java

```diff
@@ -114,7 +128,112 @@
 
         typeMap.put("Template", TypeUtil.createObjectType("Template", templateFields));
 
+        final Map<String, TypeFetcher> viewAsFields = new HashMap<>();
+        viewAsFields.put("visitor", new TypeFetcher(GraphQLTypeReference.typeRef("Visitor"),
+                new PropertyDataFetcher<ViewAsPageStatus>("visitor")));
+        viewAsFields.put("language", new TypeFetcher(GraphQLTypeReference.typeRef("Language"),
+                new LanguageDataFetcher()));
+        viewAsFields.put("mode", new TypeFetcher(GraphQLString, new PageModeDataFetcher()));
+        viewAsFields.put("persona", new TypeFetcher(GraphQLTypeReference.typeRef("PersonaBaseType"),
```

Comment:
Codacy found an issue: [The String literal "persona" appears 5 times in this file; the first occurrence is on line 137](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6028986)